modules/search_api_solr_autocomplete/src/Plugin/search_api_autocomplete/suggester/Spellcheck.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
We should not remove the old plugin because we might break existing sites. Also there might be custom code modules extending this class. If we remove it we break their site. Instead, we should deprecate it, see https://www.drupal.org/about/core/policies/core-change-policies/drupal-deprecation-policy#how-plugin to find out how. This is true for all plugins we're moving.

Reply:
This was already agreed with @mkalkbrenner and @saidatom to keep the classes as deprecated but remove the annotation, so that we can keep the current plugin IDs.